### PR TITLE
add Card component

### PR DESCRIPTION
**Pull request by gadgetfather**
I have created a card component. please review them. ( in this branch I have not added button.css files that's why buttons have a default style, so when I merge it will have styling) Check the layout, please!

preview link: https://vcqfx.csb.app/card/card.html

image
image
image
image

### CODE REVIEW

File: card/card.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+.horz-img{
+    width: 30%;
+    height: auto;
+}
+
+/* card badge */
+.card-badge{
+    width: 3rem;
+    /* height: 3rem; */
```
Comment (shaik-saad) [inline card/card.css:60]: You can remove this if this does not need to be here.

**Review by shaik-saad — APPROVED** [review]
Looks very clean Akshay, added a review other than that you're good to merge.

File: card/card.css (hunk 1 above)
Comment (gadgetfather) [reply card/card.css:60]: yea i will do it.

**Review by b0llu — APPROVED** [review]
Great work Akshay! code is clean and looks good as well.

**Review by Aazam10 — APPROVED** [review]
Looks good!

**Pull request merged**